<launch>

  	<arg name="simulation"			default='false'/>
	<arg name="min_dist_to_tower"	default='0.1'/> 
	<arg name="goal_topic"			default='/game/goal'/>
	<arg name="block_timeout"		default='3.0'/>
	<arg name="robot_base"			default='robot_0/base_link'/>
	<arg name="player_base"			default='robot_1/base_link'/>
	
	
	<node name="steering_behavior_node" pkg="steering_behavior" type="steering_behavior_node" output="screen" respawn="true">
		 <rosparam command="load" file="$(find steering_behavior)/config/params.yaml"/>
		 <param name="simulation" 		 type="bool" value="$(arg simulation)"/>
		 <param name="min_dist_to_tower" type="double" value="$(arg min_dist_to_tower)"/>
		 <param name="goal_topic" 		 type="str" value="$(arg goal_topic)"/>
		 <param name="block_timeout" 	 type="double" value="$(arg block_timeout)"/>
		 <param name="robot_base" 	 	 type="str" value="$(arg robot_base)"/>
		 <param name="player_base" 	 	 type="str" value="$(arg player_base)"/>
	</node>

</launch>